checkST2110.js

```diff
@@ -1401,6 +1494,83 @@ const test_22_74_1 = (sdp, params) => {
   return errors;
 };
 
+const ssnPermitted40 = ['ST2110-40:2018', 'ST2110-40:2021'];
+
+// ST 2110-40:2023 Section 7 Test 1 - Check SSN is the required fixed value
+const test_40_2023_7_1 = (sdp, params) => {
+  let [mtParams, errors] = extractMTParams(sdp, params);
+  for (let stream of mtParams) {
+    if (typeof stream.SSN !== 'undefined') {
+      if (ssnPermitted40.indexOf(stream.SSN) < 0) {
+        errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'SSN' is not set to the required value as per ST 2110-40:2023 Section 7.`));
+      } else {
+        let detect2023tm = typeof stream.TM !== 'undefined';
+
+        if (detect2023tm && stream.SSN === 'ST2110-40:2018') {
+          errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'SSN' is not set to the required value as per ST 2110-40:2023 Section 7.`));
+        }
+        if (!detect2023tm && stream.SSN !== 'ST2110-40:2018') {
+          errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'SSN' is not set to the required value as per ST 2110-40:2023 Section 7.`));
+        }
+      }
+    }
+  }
+  if (params.verbose && errors.length == 0) {
+    console.log('Test Passed: ST 2110-40:2023 Section 7 Test 1 - SSN is the required fixed value.');
```

Comment:
```suggestion
    console.log('Test Passed: ST 2110-40:2023 Section 7 Test 1 - SSN if present is the required fixed value.');
```

Unless we tighten up the test to check if SSN is actually present then can leave this as is.